packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -145,7 +146,7 @@ export function useContest() {
       setEntryCharge(null);
     }
 
-    if (version >= 4.2) {
```

Comment:
This is another case of this issue which results in `sortingEnabled` not being set even when it should on any contracts between 4.10 and 4.19 inclusive.